packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -0,0 +1,655 @@
+      includeFavorite: false,
+      includeTrackPage: false,
+      isArtistPick,
+      includeEmbed: !(isUnlisted || isPremium),
+      includeArtistPick: !isUnlisted,
+      includeAddToPlaylist: !(isUnlisted || isPremium),
+      extraMenuItems: overflowMenuExtraItems
+    }
+  }
+
+  const fadeIn = {
+    [styles.show]: !isLoading,
+    [styles.hide]: isLoading
+  }
+
+  return (
+    <Tile
+      className={styles.giantTrackTile}
+      dogEar={dogEarType}
+      size='large'
+      elevation='mid'
+    >
+      <div className={styles.topSection}>
+        <GiantArtwork
+          trackId={trackId}
+          coverArtSizes={coverArtSizes}
+          coSign={coSign}
+          callback={onArtworkLoad}
+        />
+        <div className={styles.infoSection}>
+          <div className={styles.infoSectionHeader}>
+            {renderCardTitle(cn(fadeIn))}
+            <div className={styles.title}>
+              <h1 className={cn(fadeIn)}>{trackTitle}</h1>
+              {isLoading && <Skeleton className={styles.skeleton} />}
+            </div>
+            <div className={styles.artistWrapper}>
+              <div className={cn(fadeIn)}>
+                <span>By</span>
+                <ArtistPopover handle={artistHandle}>
+                  <h2 className={styles.artist} onClick={onClickArtistName}>
+                    {artistName}
+                    <UserBadges
+                      className={styles.verified}
+                      badgeSize={18}
+                      userId={userId}
+                    />
+                  </h2>
+                </ArtistPopover>
+              </div>
+              {isLoading && (
+                <Skeleton className={styles.skeleton} width='60%' />
+              )}
+            </div>
+          </div>
+
+          <div className={cn(styles.playSection, fadeIn)}>
+            <PlayPauseButton
+              doesUserHaveAccess={doesUserHaveAccess}
+              playing={playing}
+              onPlay={onPlay}
+              trackId={trackId}
+            />
+            {isLongFormContent && isNewPodcastControlsEnabled ? (
+              <GiantTrackTileProgressInfo
+                duration={duration}
+                trackId={trackId}
+              />
+            ) : (
+              renderListenCount()
+            )}
+          </div>
+
+          <div className={cn(styles.statsSection, fadeIn)}>
+            {renderStatsRow()}
+          </div>
+
+          <div
+            className={cn(styles.actionButtons, fadeIn)}
+            role='group'
+            aria-label={messages.actionGroupLabel}
+          >
+            {renderShareButton()}
+            {renderMakePublicButton()}
+            {doesUserHaveAccess && renderRepostButton()}
+            {doesUserHaveAccess && renderFavoriteButton()}
+            <span>
+              {/* prop types for overflow menu don't work correctly
+              so we need to cast here */}
+              <Menu {...(overflowMenu as any)}>
```

Comment:
Didn't want to go down a rabbit hole of trying to fix the props definition for Menu... 😢